From 863eb9e2ccce6d28d4411749a4e6864d4b78068e Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:37:50 -0400
Subject: [alsa] echoaudio: Remove unused variable

Message-id: <140972823573-373-patchset-jkysela@redhat.com>
Patchwork-id: 91069
O-Subject: [RHEL 7.1 PATCH 372/442] ALSA: echoaudio: Remove unused variable
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

'chip' is not used in the function. Remove it.

Signed-off-by: Sachin Kamat <sachin.kamat@samsung.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 427f42e4cf3a681d181fd1f0492731b167807877)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/echoaudio/echoaudio.c b/sound/pci/echoaudio/echoaudio.c
index 9f10c9e..631aaa4 100644
--- a/sound/pci/echoaudio/echoaudio.c
+++ b/sound/pci/echoaudio/echoaudio.c
@@ -1754,9 +1754,6 @@ static struct snd_kcontrol_new snd_echo_vumeters_switch = {
 static int snd_echo_vumeters_info(struct snd_kcontrol *kcontrol,
       struct snd_ctl_elem_info *uinfo)
 {
- struct echoaudio *chip;
-
- chip = snd_kcontrol_chip(kcontrol);
  uinfo->type = SNDRV_CTL_ELEM_TYPE_INTEGER;
  uinfo->count = 96;
  uinfo->value.integer.min = ECHOGAIN_MINOUT;
@@ -1798,9 +1795,6 @@ static struct snd_kcontrol_new snd_echo_vumeters = {
 static int snd_echo_channels_info_info(struct snd_kcontrol *kcontrol,
            struct snd_ctl_elem_info *uinfo)
 {
- struct echoaudio *chip;
-
- chip = snd_kcontrol_chip(kcontrol);
  uinfo->type = SNDRV_CTL_ELEM_TYPE_INTEGER;
  uinfo->count = 6;
  uinfo->value.integer.min = 0;
-- 
1.7.1